packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -906,6 +902,22 @@ export function TransferPanelMain({
                     forToken={selectedToken}
                     prefix={selectedToken ? 'BALANCE: ' : ''}
                   />
+                  {app.isDepositMode &&
```

Comment:
It would be nice to extract this big conditional into a variable. Looks like `showUSDCNotice` can be repurposed, too. Maybe something like `isTransferToNativeUSDCSupportedNetwork`